cmake build step fails in Linux

### Symptom
`cmake` step fails with an error indicating that a source file is missing:

```
vagrant@vagrant:~/rdfs/build$ cmake ..
-- The C compiler identification is GNU 5.4.0
-- The CXX compiler identification is GNU 5.4.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Looking for pthread_create
-- Looking for pthread_create - not found
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - found
-- Found Threads: TRUE  
-- Found Protobuf: /usr/lib/libprotobuf.so  
-- Boost version: 1.58.0
-- Found the following Boost libraries:
--   system
-- Found GTest: /usr/lib/libgtest.a  
-- Configuring done
CMake Error at rice-namenode/CMakeLists.txt:27 (add_library):
  Cannot find source file:

    source/HaServiceProtocolImpl.cc

  Tried extensions .c .C .c++ .cc .cpp .cxx .m .M .mm .h .hh .h++ .hm .hpp
  .hxx .in .txx


-- Generating done
-- Build files have been written to: /home/vagrant/rdfs/build

```

### Reproduction steps
1. On Linux filesystem (ext4), pull `c554a6f8e9f6fe3dc937f9768d5dad7e36538c61`
2. `mkdir build`
3. `cd build && cmake ..`
4. Observe exit nonzero

### Root cause
Seems to be caused by the fact that ext4 is case sensitive, while OS X/Windows default filesystems are not.

--- a/rice-namenode/CMakeLists.txt
+++ b/rice-namenode/CMakeLists.txt
@@ -24,7 +24,7 @@ add_executable(namenode source/main.cc )
 
 target_link_libraries(namenode namenode_protos)
 
-add_library(ha_protos source/HaServiceProtocolImpl.cc )
+add_library(ha_protos source/HAServiceProtocolImpl.cc )
 target_link_libraries( ha_protos rpcserver )
 target_link_libraries( ha_protos zkwrapper )
 target_link_libraries( ha_protos rpcserver )

